pkg/connector/helpers.go

```diff
@@ -23,3 +28,17 @@ func WithRateLimitAnnotations(
 
 	return outputAnnotations
 }
+
+// shouldIncludeUser only include extant, human users.
+func shouldIncludeUser(ctx context.Context, user client.ConfluenceUser) bool {
+	logger := ctxzap.Extract(ctx)
+	if user.AccountType != accountTypeAtlassian {
+		logger.Debug("confluence: user is not of type atlassian", zap.Any("user", user))
+		return false
+	}
+	if len(user.Operations) == 0 {
```

Comment:
We're not expanding operations in the list users call, so we don't add any users.

GetUsersFromSearch() needs to add `"expand": "operations",` to its query params.